sale_commission_formula/models/sale_commission_line_mixin.py

```diff
@@ -0,0 +1,29 @@
+#  -*- coding: utf-8 -*-
+#  Copyright 2019 Simone Rubino - Agile Business Group
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+from odoo.tools.safe_eval import safe_eval
+
+
+class SaleCommissionLineMixin(models.AbstractModel):
+    _inherit = 'sale.commission.line.mixin'
+
+    @api.model
+    def _get_formula_input_dict(self):
```

Comment:
Again, this shouldn't be needed

Reply:
I'm sorry but I'm not getting what you mean here.
The method is created to be overridden from whoever might be willing to add values for the formula to use. 
I can't find any other occurrence of `_get_formula_input_dict` so it is not overriding anything

Reply:
On a side note, I just edited that to use the correct decorator `api.multi` with `ensure_one`

Reply:
OK, got it. I thought this method was declared already on super. My fault.